### ISSUE TITLE
Feature Request: Better output format

### ISSUE THREAD

**Issue by agraubert** [broadinstitute/canine#5]
Orchestrator should return output in a better format (big dataframe) rather than a bunch of separate outputs

**Comment by agraubert** [comment]
I'm thinking a pandas dataframe:

```
JOB_ID JOB_ALIAS INPUT_1 INPUT_2 INPUT_N OUTPUT_1 OUTPUT_N EXIT_CODE SLURM_JOB_STATUS```

**Comment by agraubert** [comment]
closed in 63cc655e2cf399704247cbbca46c5b6ea7636fb8
